package.json

```
@@ -72,7 +73,7 @@
       ]
     }
   },
-  "module": "./fake-timers.js",
+  "module": "./fake-timers-esm.js",
```

After this change, the produced bundle is not being referenced anywhere. Let's not forget it, aligning it with Sinon's package.json.

```suggestion
  "module": "./pkg/fake-timers-esm.js",
  "cdn": "./pkg/fake-timer.js",
  "jsdelivr": "./pkg/fake-timer.js",
```

Here I updated the paths. I'll add some commentary in my general feedback.